.ado/scripts/waitForVerdaccio.mjs

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+// @ts-check
+
+import * as fs from "node:fs";
+import { get } from "node:http";
+
+/**
+ * @param {string} npmRegistryUrl
+ */
+function queryForServerStatus(npmRegistryUrl) {
```

Comment:
Just double checking, this is new code that isn't a port of an older file, right?

Reply:
NVM. I see the old one.